staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2191,17 +2191,41 @@ type SMBiosConfiguration struct {
-	BandwidthPerMigration             *resource.Quantity `json:"bandwidthPerMigration,omitempty"`
-	CompletionTimeoutPerGiB           *int64             `json:"completionTimeoutPerGiB,omitempty"`
-	ProgressTimeout                   *int64             `json:"progressTimeout,omitempty"`
-	UnsafeMigrationOverride           *bool              `json:"unsafeMigrationOverride,omitempty"`
-	AllowPostCopy                     *bool              `json:"allowPostCopy,omitempty"`
-	DisableTLS                        *bool              `json:"disableTLS,omitempty"`
-	Network                           *string            `json:"network,omitempty"`
+	// NodeDrainTaintKey defines the taint key that indicates a node should be drained.
+	// Note: this option relies deprecated node taint feature. Default: kubevirt.io/drain
+	NodeDrainTaintKey *string `json:"nodeDrainTaintKey,omitempty"`
+	// ParallelOutboundMigrationsPerNode is the maximum number of concurrent outgoing live migrations
+	// allowed per node. Defaults to 2
+	ParallelOutboundMigrationsPerNode *uint32 `json:"parallelOutboundMigrationsPerNode,omitempty"`
+	// ParallelMigrationsPerCluster it the total number of concurrent live migrations
+	// allowed cluster-wide. Defaults to 5
+	ParallelMigrationsPerCluster *uint32 `json:"parallelMigrationsPerCluster,omitempty"`
+	// AllowAutoConverge allows the platform to compromise performance/availability of VMIs to
+	// guarantee successful VMI live migrations. Defaults to false
+	AllowAutoConverge *bool `json:"allowAutoConverge,omitempty"`
+	// BandwidthPerMigration limits the amount of network bandwith live migrations are allowed to use.
+	// The value is in MiB per second. Defaults to 0 (no limit)
+	BandwidthPerMigration *resource.Quantity `json:"bandwidthPerMigration,omitempty"`
+	// CompletionTimeoutPerGiB is the maximum number of seconds 1GiB can take to live-migrate.
+	// If, during any CompletionTimeoutPerGiB period, less than 1GiB is transfered from the source
+	// to the target, the migration is cancelled. Defaults to 800
+	CompletionTimeoutPerGiB *int64 `json:"completionTimeoutPerGiB,omitempty"`
+	// ProgressTimeout is the maximum number of seconds a live migration is allowed to idle.
+	// Defaults to 150
+	ProgressTimeout *int64 `json:"progressTimeout,omitempty"`
+	// UnsafeMigrationOverride allows live migrations to occur even if the compatibility check
+	// indicates the migration will be unsafe to the guest. Defaults to false.
+	UnsafeMigrationOverride *bool `json:"unsafeMigrationOverride,omitempty"`
+	// AllowPostCopy enables post-copy live migrations. Such migrations allow even the busiest VMIs
+	// to successfully live-migrate. However, events like a network failure can cause a VMI crash.
+	// Defaults to false
+	AllowPostCopy *bool `json:"allowPostCopy,omitempty"`
+	// When set to true, DisableTLS will disable the additional layer of live migration encryption
+	// provided by KubeVirt. Defaults to false
+	DisableTLS *bool `json:"disableTLS,omitempty"`
```

Comment:
@vladikr I remember that we talked about deprecating this one. Maybe we should say it's not encouraged?

Reply:
Added a few words to that effect, PTAL!